File: 은지/LV1/폰켓몬.py
def solution(nums):
    answer = 0
    N = len(nums) // 2
    # 폰켓몬의 종류
    mon = []
    for x in nums:
        if x not in mon:
            mon.append(x)
    # 폰켓몬의 종류 수 < 가져갈 수 있는 폰켓몬의 수
    if len(mon) < N:
        answer = len(mon)
    # 폰켓몬의 종류 수 >= 가져갈 수 있는 폰켓몬의 수
    else:
        answer = N
    return answer

File: 은지/LV1/폰켓몬.js
function solution(nums) {
  var answer = 0
  const N = nums.length / 2
  var mon = []
  for (let x of nums) {
    if (!mon.includes(x)) {
      mon.push(x)
    }
  }
  if (mon.length < N) {
    answer = mon.length
  } else {
    answer = N
  }
  return answer;
}